stan/math/rev/fun/divide.hpp

```diff
@@ -1,208 +1,8 @@
 #ifndef STAN_MATH_REV_FUN_DIVIDE_HPP
 #define STAN_MATH_REV_FUN_DIVIDE_HPP
 
+#include <stan/math/prim/meta.hpp>
 #include <stan/math/rev/meta.hpp>
 #include <stan/math/rev/core.hpp>
-#include <stan/math/rev/fun/to_var.hpp>
```

Comment:
Should this file be completely empty? Should it just be removed?

Reply:
I think for `rstan` we want to be careful of deleting files. idt there's anything wrong with leaving this here for now